great_expectations/checkpoint/checkpoint.py

```diff
@@ -250,8 +250,11 @@ def _run_validation(
         result_format: Optional[dict],
         run_id: Optional[Union[str, RunIdentifier]],
         idx: Optional[int] = 0,
-        validation_dict: Optional[dict] = {},
+        validation_dict: Optional[dict] = None,
```

Comment:
Why not make it not optional and default to {}?

Reply:
@donaldheppner This is important: putting defaults in the arguments is a mutability hazard. We must avoid this practice. Thanks.